internal/cli/users_blocks.go

```diff
@@ -25,32 +26,39 @@ func userBlocksCmd(cli *cli) *cobra.Command {
 
 func listUserBlocksCmd(cli *cli) *cobra.Command {
 	var inputs struct {
-		userID string
+		userIdentifier string
 	}
 
 	cmd := &cobra.Command{
 		Use:   "list",
 		Args:  cobra.MaximumNArgs(1),
 		Short: "List brute-force protection blocks for a given user",
-		Long:  "List brute-force protection blocks for a given user.",
-		Example: `  auth0 users blocks list <user-id>
-  auth0 users blocks list <user-id> --json`,
+		Long:  "List brute-force protection blocks for a given user by user ID, username, phone number or email.",
+		Example: `  auth0 users blocks list <user-id|username|email|phone-number>
+  auth0 users blocks list <user-id|username|email|phone-number> --json
+  auth0 users blocks unblock "auth0|61b5b6e90783fa19f7c57dad
+  auth0 users blocks unblock "frederik@travel0.com`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 0 {
-				if err := userID.Ask(cmd, &inputs.userID); err != nil {
+				if err := userID.Ask(cmd, &inputs.userIdentifier); err != nil {
 					return err
 				}
 			} else {
-				inputs.userID = args[0]
+				inputs.userIdentifier = args[0]
 			}
 
 			var userBlocks []*management.UserBlock
 			err := ansi.Waiting(func() (err error) {
-				userBlocks, err = cli.api.User.Blocks(cmd.Context(), inputs.userID)
+				userBlocks, err = cli.api.User.Blocks(cmd.Context(), inputs.userIdentifier)
+				if mErr, ok := err.(management.Error); ok && mErr.Status() != http.StatusBadRequest {
+					return nil
+				}
+
+				userBlocks, err = cli.api.User.BlocksByIdentifier(cmd.Context(), inputs.userIdentifier)
 				return err
```

Comment:
This looks like it implicitly returns one of the possible two errors: it could be the first error, the second error, or nil. Should we check for the first error and shortcut the process if it exists?